### DISCUSSION TITLE
苦求一款合适的低代码平台

### DISCUSSION THREAD

**Discussion by dxdingdu** [taowen/awesome-lowcode#104] [Q&A]
多年寻觅而不得。我们需要的不是花里胡捎的界面表单、h5，我们要的是数据据查询分析器的替代功能，让普通员工有能力维护数据，而不用亲自开发一大堆列表、表单系统。

具体需求：
1、支持mysql
2、基于我们已有的业务数据表，快速构建列表、详情界页，支持增删改查
3、支持各种常见字段类型，下拉框支持定义常量代码
4、支持单点登录
5、操作简单

各位大佬有推荐的吗

**Comment by li-ml** [comment]
一般这类软件要收费的，没有太多开源的

**Reply by dxdingdu** [reply]
收费能接受，大佬有推荐吗

**Reply by lesliechueng** [reply]
> 收费能接受，大佬有推荐吗

我能提供一套完整强大的低代码平台，想了解可以扫微信
![a6fc555f00c7df7942bc0432c330d71](https://user-images.githubusercontent.com/13750154/222945659-dea272bb-d66c-4bbb-8a03-0936de7608bd.jpg)

**Comment by ghost** [comment]
钉钉宜搭  付费升级后可以直连mysql，做增删改查页面很简单，订阅制付费，适合本来就在用钉钉的
如果公司用企业微信，腾讯云微搭应该也可以
活字格，直连mysql，做增删改查页面很简单，按账号数量一次性付费，永久使用，但是单价比较贵，可以单点登录

**Comment by yangaijun** [comment]
light2f.com 看看能不能满足你的需求

**Comment by LowCoderGitHub** [comment]
我平台上有很多低代码平台的介绍和讨论 可以来这里了解一下
https://www.lowcoder.com.cn/
